supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -168,6 +168,26 @@ If referring to the program, use "Emacs", for example, "Source-Navigator support
 
 *See also*:
 
+[[executable]]
```

Comment:
This produces a warning about duplicate IDs in use. Can you change this one to add -adj?

```suggestion
[[executable-adj]]
```